content/en/community/events.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Events
+cascade:
+  type: docs
+---
+
+The OpenTelemetry community participates in and organizes various events throughout the year to bring together contributors, maintainers, and end users. These events provide opportunities to learn, share experiences, collaborate on the project, and connect with the broader observability community.
+
+## Upcoming Events
+
+{{< community-event-list >}}
+
+You can also find the latest event announcements in our [RSS feed](/index.xml) and [announcements section](/announcements/).
+
+## Key Community Events
+
+### KubeCon + CloudNativeCon
+
+OpenTelemetry has a strong presence at KubeCon + CloudNativeCon events worldwide. These events feature:
+
+- OpenTelemetry maintainer talks and workshops
+- Community meetups at the OpenTelemetry Observatory
+- Hands-on sessions and tutorials
+- Networking opportunities with contributors and end users
+
+### OpenTelemetry Community Day
+
+OpenTelemetry Community Day is a dedicated event focused entirely on OpenTelemetry, featuring:
+
+- Deep-dive technical sessions
+- End user case studies and success stories
+- Hands-on workshops and tutorials
+- Community roadmap discussions
+- Networking with maintainers and contributors
+
+Check the [CNCF Events page](https://events.linuxfoundation.org/) for the latest OpenTelemetry Community Day announcements.
```

Comment:
I would assume this link also works for KubeCon & CloudNativeCon and Observability Summit?